WasatchNET/EEPROM.cs

```diff
@@ -1299,7 +1305,7 @@ public bool read()
                     string test = buffer.ToString();
 
                    
-                    wavecalCoeffs = new float[] { 0, 1, 0, 0 };
+                    wavecalCoeffs = new float[] { 0, 1, 0, 0 }; // MZ: add a zero?

```

Comment:
I'm unsure on this one. My thinking as I added support for 4th order was "we support it but only if explicitly asked for," I personally want to think on it a little more but having the 0 here may very well make sense